### ISSUE TITLE
Support ActiveMQ for use with OpenShift

### ISSUE THREAD

**Issue by docktermj** [Senzing/community-roadmap#4]
**Is your feature request related to a problem? Please describe.**
Currently ActiveMQ is not supported.

**Describe the solution you'd like**
Add support for ActiveMQ.

**Comment by docktermj** [comment]
Supporting issues:

- https://github.com/Senzing/stream-producer/issues/29
- https://github.com/Senzing/stream-loader/issues/142
- https://github.com/Senzing/redoer/issues/32
- https://github.com/Senzing/stream-logger/issues/17

**Comment by jeffadair** [comment]
I've started to look into this, and I'm going to recommend holding off an encouraging customers to use other messaging systems (RabbitMQ or SQS depending). As I've looked into it, I've run into a couple hurdles that make me feel like this is going to be more difficult to implement and support that other methods.

1. There is no "official" activeMQ broker docker image from Apache, or from our normal sources like Bitnami. The broker I am using is from [rmohr on dockerhub](https://hub.docker.com/r/rmohr/activemq/). It has been downloaded over 10M times, but has not been updated for over a year. I question the health of this project since it's not backed by an organization. It does work well. 
2. There are [two methods that Apache recommends](http://activemq.apache.org/python.html) for python integration. The wrapper around the C++ libraries is not supported anymore. The other is to find a STOMP library. There are only two that support the latest version of activeMQ and the most recommended option (stomp.py) has some unusual quirks that may require us to fork it.

There is also ActiveMQ Artemis which is supposed to be an eventual successor to ActiveMQ. I was unable to get their official docker  broker to work for me.

All this does not give me much confidence in ActiveMQ or Artemis, thus why I suggest recommending other methods.
